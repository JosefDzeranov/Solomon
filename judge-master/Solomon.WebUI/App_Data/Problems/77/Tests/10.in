nnopojippsjhwtcngbloykcwvsndcssgfwd zdzrwxiajbknmwscqjsvjhdxshzhapntcfr 1 xgmjfxkjyvyqu 1 xjtqqrabzpybauorkhfnfkwipmwtzxiimvsdalpz 1 whwldxqv zvu qoynuodqstnqwmyzygoudsyruq 1 uovvaugainvtgnobtuhwhcrsthabypgtdkic  1 ofstdawiqmt 1 ttoxpbkrgxxcgjxjxh 1 x 1 mubihzkqpw 1 kztnvaypk 1 xqhpnucnkjivahzmohczaratshpjo dzxlogytnrxfwfrpll 1 xu 1 pvptokpkzfyq vxncaijxltouhwhjymapdfvmggt 1 oajvziswiocszxmlxkyxibpilqtqbbvup 1 kjtvydkakjiqfsjpvhvwnlaj flynkwlqzpknqjvzsfvmkpgbazqomh 1 p 1 oxamdpqpprxkglkzkfqk ucltyxsbgyliwydootci 1 wb 1 qslj 1 voaioghjl 1  giakvhqpvwokdqlyagcfliwk ynvpmpmcmhhorfxmuwswcbtufcthcqv 1 l 1 uhlgvmfqgzsmhjigjnqgtwuakjwfawmcdmkqanbfqmutbtvjywgh qxmsydh 1 s 1 qqdlodvaxlo
dsxjusnpcohxkjyowrvwoyawlxhin xftfpdorniabkazmiobyxgjtcwnfbysyqiiqpdaibyrmuthdaxscoxtrvadtpjpppc vxncaijxltouhwhjymapdfvmggt 1 ajvziswiocszxmlxkyxibpilqtqbbvup 1 ikjtvydkakjiqfsjpvhvwnlaj vqnlp zdzrwxiajbknmwscqjsvjhdxshzhapntcfr 1 xgmjfxkjyvyqu 1  jtqqrabzpybauorkhfnfkwipmwtzxiimvsdalpz 1 whwldxqv