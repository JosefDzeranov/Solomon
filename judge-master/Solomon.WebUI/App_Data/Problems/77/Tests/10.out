ndsjfwyofxncpcoqvkxolwfjqq ncvynlgbsjvzlaabwpkt kcimq osjvfdottxaalijbnggc1 tgud pshykxuudhyr jvvlbli yhjtl ogdqwqdha pa1xzuaak  ncnbo jfxuivsww1kt lipjzz1gvqqtd iws p yhi  sxts  qk ipvgvv pdh1mzrcqx1huow1fofwam tja p z wvurm  p ui lmqbkp1wyx szhxtuqst1xj1hokyhk vm uwl jdajz  t  qo wcjn  1hclago hzptxq1h1mh phstk1u qm kh  wrnqio a updvjzvw cqph1j   twtqiyubtbnzpyxylplsvh wq  cxcrmnoytiuxtmmdq tlwoufx  nifavuvpohcloalkz1yjorham  garbsovgxznokpxap x kflws  bj zddatpkkgpdkkkos1dxgmy  lb1paqudbqjykfyjnxb qmvcd  ok ylsgkkpitzvxiqagvlumdh  ynxbptairwvnfmiqjmyoywfm   kmgaznicg arygbfvdlaasqk1  cwmu qn x1hxqgpszpiigwgq   wsjo1wv x zf tijsqwocczas  vcfr mt1ckmwv lpfpygfbsn   sqxkwyg gzofx1qvvpdhltmb1  njkhhznojthrn thmrojiuhf   
